MCprep_addon/addon_updater_ops.py

```diff
@@ -81,7 +81,11 @@ def check_for_update(self, now):
 # Blender version utils
 # -----------------------------------------------------------------------------
 def make_annotations(cls):
-	"""Add annotation attribute to fields to avoid Blender 2.8+ warnings"""
+	"""Add annotation attribute to fields to avoid Blender 2.8+ warnings.
+
+	Deprecated operator as MCprep 3.5 moves to support 2.8+ only and using
```

Comment:
@TheDuckCow just curious, does this apply to the addon updater as a whole? I was under the impression that the addon updater was independent of MCprep in terms of implementation (i.e. the addon updater is managed as a separate project)

Reply:
This is only true inside the addon updater since it's still self-annotating. And yes, the addon udpater comes from here: 
https://github.com/CGCookie/blender-addon-updater

Hindsight is 20:20, I'm wishing that when I built the addon updater for CGCookie that I had set out from the getgo to have it setup as a pip installable. Unfortunately, there's slight code changes developers need to make, which make updating it also a challenge. So I'd advise keeping the updater mostly as-is for now.